testutil/singlevm/setup.sh

```diff
@@ -207,6 +214,27 @@ sudo cp -f clear-"${LATEST}"-cloud.img /var/lib/ciao/images
 cd /var/lib/ciao/images
 sudo ln -sf clear-"${LATEST}"-cloud.img df3768da-31f5-4ba6-82f0-127a1a705169
 
+# Set macvlan interface
+if [ -x "$(command -v ip)" ]; then
+    sudo ip link add name eth10 type bridge
```

Comment:
@obedmr can we delete eth10 before we add it, to handle the issue where the user terminates the script midway with a ctrl C